### ISSUE TITLE
Longer videos fail 1080p transcoding and get stuck at 360p max resolution

### ISSUE THREAD

**Issue by lucky1804** [mediacms-io/mediacms#1098]
**Describe the issue**
So testing with exact same codec I was able to upload multiple 1 minute 1080p60fps videos and multiple 5 minute 1080p60fps videos without any issue but when trying to upload longer videos (for example 8 minutes or longer) at 1080p60fps the videos start of as usual with only 240p and 360p being available then after few minutes videos disappear again replaced by media being encoded messages before coming back few minutes later available at only 360p resolution.
Checking Django panel most resolutions show fail BUT still have path to fully encoded video that shows no issue, 1080p reoslution is the only one that actually doesnt have encoded file.

**To Reproduce**
Steps to reproduce the issue:
1. Upload 10 minute 1080p60fps video
2. Wait for transcoding to be done
3. Video is on available at 360p resolution

**Expected behavior**
Just like with shorter videos I expect longer videos to be fully transcoded too.

**Screenshots**

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04.05 LTS in Virtualbox
 - Installation method: Single server in Virtualbox

**Additional context**
I was testing Mediacms about two or three months ago with EXACT same setup (Ubuntu in Virtualbox) and was able to transcode 1440p video without issues but now that I reinstalled it it wont transcode 1080p60 videos if its longer than maybe like 5 minutes.

**Comment by youssefmaher99** [comment]
I am facing the same issue

**Comment by lucky1804** [comment]
> I am facing the same issue

I just tried again and it has not been fixed yet.

**Comment by foeyonghai** [comment]
@mgogoulos do you have an idea of how needed it the transcode function and if it still should be enabled by default? It required a lot of CPU power and hard disk space and the question is how many people really have a benefit from it.

**Comment by foeyonghai** [comment]
As you can see above, just created a feature request (it's just a setting to be changed and that's all). https://github.com/mediacms-io/mediacms/issues/1126.

**Comment by mgogoulos** [comment]
@foeyonghai I'll check the PR when I have some time and write you there if it needs something, otherwise merge it. 

As far as whether transcoding is needed or not, I'd say it's necessary for the project, for the following reasons:
1. it transcodes videos to smaller sizes, so they are possible to play through the web. For a local installation you can watch original videos, but if it passes through the internet, chances are you need smaller versions
2. it standarizes to formats that all players can play, so no matter what video type you upload, it gets transcoded to something that all players can play. 
I'd say that the vast majority of the portals created with mediaCMS gain value by the fact that there's an out of the box experience that works and can help share video and other media content. 
But since we see value on just showing the original ones, it's a nice feature to include this too.

**Comment by lucky1804** [comment]
> @mgogoulos do you have an idea of how needed it the transcode function and if it still should be enabled by default? It required a lot of CPU power and hard disk space and the question is how many people really have a benefit from it.

For my use case transcoding is needed and I really appriciate that its enabled out of the box without any additional setup, if you dont need it you can disable it.

**Comment by lucky1804** [comment]
> @foeyonghai I'll check the PR when I have some time and write you there if it needs something, otherwise merge it.
> 
> As far as whether transcoding is needed or not, I'd say it's necessary for the project, for the following reasons:
> 
> 1. it transcodes videos to smaller sizes, so they are possible to play through the web. For a local installation you can watch original videos, but if it passes through the internet, chances are you need smaller versions
> 2. it standarizes to formats that all players can play, so no matter what video type you upload, it gets transcoded to something that all players can play.
>    I'd say that the vast majority of the portals created with mediaCMS gain value by the fact that there's an out of the box experience that works and can help share video and other media content.
>    But since we see value on just showing the original ones, it's a nice feature to include this too.

Yes, transcoding is really needed when you have multiple people with different devices and browser running over the internet but at the moment transcoding doesnt work for any larger videos as mentioned here too https://github.com/mediacms-io/mediacms/issues/1121

Is there any fix for this at the moment?

**Comment by mgogoulos** [comment]
This should be fixed now on the latest version 4.2, can you update and test? Also check https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md  , you might have to restart some services

**Comment by lucky1804** [comment]
> This should be fixed now on the latest version 4.2, can you update and test? Also check https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md , you might have to restart some services

I will test it in the next few days and I will let you know if issue is fixed.
